README.md

```diff
@@ -13,6 +13,34 @@ The __filter__ package is generally provided as a folder with source (`.java`) a
 
 A sample set of images are provided in an __"images"__ directory.
 
+## Creating an executable jar file
+The project is already set up using the gradle wrapper so we can just use that to create the jar file.
+1. Open up the CLI and change the directory to the projects root directory.
+2. Execute the command `./gradlew build`
+3. This will build/rebuild the jar file using all the required dependencies.
+4. The new jar file will be named "imagelab.jar" and it will be located in the projects root directory.
+
+More information on gradle can be found [here](https://docs.gradle.org/current/userguide/userguide.html) and more information on the gradle wrapper can be found [here](https://docs.gradle.org/current/userguide/gradle_wrapper.html).
+
+If there is no automated build environment set up, the jar file can be created using `jar`.
+1. Open up the CLI and change the directory to the projects root directory.
+2. Execute the command `javac Run.java` to compile the Run.java file.
+3. Execute the command `javac *.java` in the package directories to compile/recompile those package java files. The package directories for this project will be the sound and imagelab directories.
+4. In the root directory execute the command `echo Main-Class: Run > MANIFEST.MF` to create the manifest file you will use for making the jar.
+5. Create the jar using the command `jar cfm DESIRED_FILENAME.jar MANIFEST.MF *.class sound/*.class imagelab/*.class`
+6. The new jar file will retain the name you gave it and will be located in the projects root directory.
```

Comment:
There's no formatting distinction between DESIRED_FILENAME.jar and MANIFEST.MF but (I believe) they have different meanings: the former is expected to be replaced and the latter is meant to be used literally.

To avoid confusion, I suggest changing `DESIRED_FILENAME.jar` to `imagelab.jar`, which is consistent with the automated build product and leaves no ambiguity.

Line 6 can now read the same as the automated build's line 4:
 ` `  The new jar file will be named "imagelab.jar" and it will be located in the project's root directory.